themes/socialbase/templates/field/field.html.twig

```diff
@@ -41,31 +41,41 @@
 
 {% if bare %}
   {% for item in items %}
-    {% if entity_type == "node" and field_name == "body" and part_of_teaser %}
+    {% if label_hidden %}
+      {% if entity_type == "node" and field_name == "body" and part_of_teaser %}
         {{ item.content|render|striptags }}
+      {% else %}
+        {{ item.content }}
+      {% endif %}
     {% else %}
-      {{ item.content }}
+      <div{{ title_attributes.addClass(title_classes) }}>{{ label }}</div>
+
+      {% if entity_type == "node" and field_name == "body" and part_of_teaser %}
+        {{ item.content|render|striptags }}
+      {% else %}
+        {{ item.content }}
+      {% endif %}
```

Comment:
This duplicates quite a bit of code. We should just render the label if it's not hidden and deduplicate the rest.

Reply:
Fixed